types/dataframe-js/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for dataframe-js 1.0
```

Comment:
```suggestion
// Type definitions for dataframe-js 1.4
```
[the published version seems to be 1.4.4:
](https://www.npmjs.com/package/dataframe-js)